llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -134,6 +181,18 @@ def gen_rotary_embedding(
                 device=
                 'cpu',  # FSDP does not materialize modules with meta buffers, hence device is set to cpu
```

Comment:
Can you also swap out the `dynamic`, `linear`, and `no_scaling` rope modifications to go through LlamaRotaryEmbedding and PartialLlamaConfig as well? transformers is deprecating the old code path